onnxruntime/core/framework/provider_bridge_ort.cc

```diff
@@ -593,14 +927,15 @@ struct ProviderSharedLibrary {
 
 static ProviderSharedLibrary s_library_shared;
```

Comment:
Is it possbile to let OrtEnv manage these static varibles? I worry the destruct order of s_library_shared/s_library_xxx and something else.

Reply:
Fear not, they might be static variables but they are destroyed by calling 'UnloadSharedProviders()' to guarantee the destruction order. Their destructors do nothing.